Creds_N_Shell_AV_Update/landing_page/login

```diff
@@ -0,0 +1,136 @@
+<!--This version of the anti-virus checking page both collects credentials then at 75%, redirects

```

Comment:
Same thing here, would be better to use a Jinja comment so the client can't see it.